vnext/DeforkingPatches/ReactCommon/yoga/yoga/Yoga.h

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE
+ * file in the root directory of this source tree.
+ */
+#pragma once
+
+#include <assert.h>
+#include <math.h>
+#include <stdarg.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+#include "YGEnums.h"
+#include "YGMacros.h"
+#include "YGValue.h"
+
+YG_EXTERN_C_BEGIN
+
+typedef struct YGSize {
+  float width;
+  float height;
+} YGSize;
+
+typedef struct YGConfig* YGConfigRef;
+
+typedef struct YGNode* YGNodeRef;
+typedef const struct YGNode* YGNodeConstRef;
+
+typedef YGSize (__cdecl *YGMeasureFunc)(
+    YGNodeRef node,
+    float width,
+    YGMeasureMode widthMode,
+    float height,
+    YGMeasureMode heightMode);
+typedef float (__cdecl *YGBaselineFunc)(YGNodeRef node, float width, float height);
+typedef void (__cdecl *YGDirtiedFunc)(YGNodeRef node);
+typedef void (__cdecl *YGPrintFunc)(YGNodeRef node);
+typedef void (__cdecl *YGNodeCleanupFunc)(YGNodeRef node);
+typedef int (__cdecl *YGLogger)(
+    YGConfigRef config,
+    YGNodeRef node,
+    YGLogLevel level,
+    const char* format,
+    va_list args);
+typedef YGNodeRef (
+    *YGCloneNodeFunc)(YGNodeRef oldNode, YGNodeRef owner, int childIndex);
+
+// YGNode
+WIN_EXPORT YGNodeRef __cdecl YGNodeNew(void);
+WIN_EXPORT YGNodeRef __cdecl YGNodeNewWithConfig(YGConfigRef config);
+WIN_EXPORT YGNodeRef __cdecl YGNodeClone(YGNodeRef node);
+WIN_EXPORT void __cdecl YGNodeFree(YGNodeRef node);
+WIN_EXPORT void __cdecl YGNodeFreeRecursiveWithCleanupFunc(
+    YGNodeRef node,
+    YGNodeCleanupFunc cleanup);
+WIN_EXPORT void __cdecl YGNodeFreeRecursive(YGNodeRef node);
+WIN_EXPORT void __cdecl YGNodeReset(YGNodeRef node);
+
+WIN_EXPORT void __cdecl YGNodeInsertChild(
+    YGNodeRef node,
+    YGNodeRef child,
+    uint32_t index);
+
+WIN_EXPORT void __cdecl YGNodeRemoveChild(YGNodeRef node, YGNodeRef child);
```

Comment:
>WIN_EXPORT [](start = 0, length = 10)

If we are already using "WIN_EXPORT" macro, perhaps we can use "WINAPIV" macro as well? So this line would be:
WIN_EXPORT void WINAPIV YGNodeRemoveChild(YGNodeRef node, YGNodeRef child);

Reply:
OK, point taken.
I'm reluctant to use Windows headers everywhere, especially if they are not needed for anything else, but it's fair to use in our patched Yoga sources.

I'll update accordingly.